server/src/com/cloud/api/ApiServlet.java

```diff
@@ -156,7 +158,7 @@ void processRequestInContext(final HttpServletRequest req, final HttpServletResp
         try {
 
             if (HttpUtils.RESPONSE_TYPE_JSON.equalsIgnoreCase(responseType)) {
-                resp.setContentType(ApiServer.getJSONContentType());
+                resp.setContentType(_apiServer.getJSONContentType());
```

Comment:
@pdube you are right, I am removing those undersocres (**_**apiServer and **_**messageBus). Thanks